spec/workers/email_worker_spec.rb

```diff
@@ -16,13 +16,13 @@
       end
     end
 
-    shared_examples "a correct address email" do |mail|
+    shared_examples "a correct address email" do |_mail|
       it "sends out email to correct address" do
         expect(@mail.to.first).to eq(user.email)
       end
     end
 
-    shared_examples "a correct subject email" do |mail, email_subject|
+    shared_examples "a correct subject email" do |_mail, email_subject|
       it "sends out email with correct subject" do
         expect(@mail.subject).to eq(email_subject)
       end
```

Comment:
I know I wrote this, but I think it's better to pass the email as an argument instead of using the class variable. Could you update that?

Reply:
Happy to, chief! But I'm not sure I got what you meant, could you clarify?